.eslintignore

```diff
@@ -0,0 +1 @@
+build/**/*.js
```

Comment:
did https://github.com/jsdrupal/drupal-admin-ui/blob/master/.gitignore#L10 not cover this?